src/pages/index.jsx

```diff
@@ -24,19 +24,14 @@ export default function Home() {
             />
           </div>
         </div>
-        <ul>
-          <li>
-            <ComponentCard name="Accordion" slug="/accordion" image="/accordion_artwork.svg" />
-          </li>
-          <li>
-            <ComponentCard name="Dialog" slug="/dialog" image="/dialog_artwork.svg" />
-          </li>
-          <li>
-            <ComponentCard name="Disclosure" slug="/disclosure" image="/disclosure_artwork.svg" />
-          </li>
-          <li>
-            <ComponentCard name="Tabs" slug="/tabs" image="/tabs_artwork.svg" />
-          </li>
+      </div>
+      <div className="components">
+        <ul className="components-grid">
+          <li className="components-grid__item components-grid__item--accordion"><ComponentCard name="Accordion" slug="/accordion" image="/accordion_artwork.svg" /></li>
+          <li className="components-grid__item components-grid__item--dialog"><ComponentCard name="Dialog" slug="/dialog" image="/dialog_artwork.svg" /></li>
+          <li className="components-grid__item components-grid__item--disclosure"><ComponentCard name="Disclosure" slug="/disclosure" image="/disclosure_artwork.svg" /></li>
+          <li className="components-grid__item components-grid__item--tabs"><ComponentCard name="Tabs" slug="/tabs" image="/tabs_artwork.svg" /></li>
+          <li className="components-grid__item components-grid__item--max-scott"><img src="/max-grid.svg" alt="Illustration of cute small white terrier dog with tongue out and head turned to the left looking like he is curious" /></li>
```

Comment:
This is excellent `alt` text!

Reply:
Thanks! As far as your advice for mobile--yes it needs to be mobile responsive so I will fix that padding as you suggested. thank you!

Reply:
Formatting note: I would expect each `ComponentCard` line to be on a new line inside of the `li` tags.

```jsx
<li>
  <ComponentCard />
</li>
```

Reply:
For the padding, I would remove the padding from the `ul`, rather than add `padding-right` to balance it out.